src/auth/auth.service.ts

```
@@ -1,145 +1,62 @@
-    const token = crypto.randomBytes(32).toString('hex');
-    const expiresAt = addMinutes(new Date(), 15);
-    const reset = this.passwordResetRepository.create({ user: user, token, expiresAt });
-    await this.passwordResetRepository.save(reset);
-    const resetLink = `https://your-app.com/reset-password?token=${token}`;
-    await this.mailService.sendEmail({
-      to: user.email,
-      subject: 'Password Reset Request',
-      body: `Click the link to reset your password: ${resetLink}`,
-    });
+  
+  async sendPasswordResetEmail(email: string): Promise<any> {
```

ESLint

[error] 'email' is defined but never used.

(@typescript-eslint/no-unused-vars)